about.html

```diff
@@ -41,14 +44,17 @@
             <div class="main-navigation">
                 <div class="logo">
                     <a href="index.html">
-                        <img src="">
+                        <img alt="Robust-Events" src="img/logo.png" class="main-logo">
```

Comment:
- Currently all your alt tags have the same description, just as the previous reviewer requested  kindly change these and add an appropriate descriptions of images in the alt tags, this makes your website more accessible and helps screen readers read your website easily 👍🏿